src/component/DetailedInfoModal.js

```diff
@@ -241,8 +248,10 @@ const DetailedInfoModal = () => {
 
     return (
         <div style={modal}>
-            <div style={blurBackground}  onClick={toggleModal}/>
-            <div style={content} className='main-background-color'>
+            <div style={blurBackground} onClick={toggleModal}/>
+            
+            <div style={content} className='main-background-color' onClick={toggleModal}>
```

Comment:
Should the onClick be here?

Reply:
Since this means clicking the modal itself closes it

Reply:
fixed the bug. It's a small mistake